assets/src/web-stories-block/block/block-types/selected-stories/storyPreview.js

```diff
@@ -17,6 +17,7 @@
  * External dependencies
  */
 import styled from 'styled-components';
+import { getRelativeDisplayDate } from '@web-stories-wp/date';
```

Comment:
Any chance we can use `@wordpress/date` to display the relative dates here?

Our blocks should ideally _not_ be using `@web-stories-wp/date`